src/ProgressBar/ProgressBar.tsx

```diff
@@ -23,6 +23,7 @@ const sizeMap = {
 type StyledProgressContainerProps = {
   inline?: boolean
   barSize?: keyof typeof sizeMap
+  tabIndex?: number
```

Comment:
Not sure if you need to add this explicitly 🤔 

If you pass tabIndex to ProgressBar, it passes it already passes it down to the html element


```jsx
<Progress tabIndex={0} />
```

Reply:
Amazing! I'll try this out.

Reply:
It looks like I need the type added:

<img width="768" alt="A screenshot of a ProgressBar storybook story. This story shows the tabIndex with a red underline indicating a typeerror." src="https://github.com/primer/react/assets/15275462/17925959-fbda-43e1-b64d-c986c1f32741">


Is there a better type to use? Maybe `React.HTMLProps` ?

Reply:
Maybe this could be an option?

```tsx
type StyledProgressContainerProps = {
  inline?: boolean
  barSize?: keyof typeof sizeMap
} & WidthProps &
  SxProp &
  React.HTMLAttributes<HTMLSpanElement>
```